src/neuroconv/tools/nwb_helpers.py

```diff
@@ -184,3 +187,19 @@ def make_or_load_nwbfile(
 
                 if not success and not file_initially_exists:
                     nwbfile_path_in.unlink()
+
+
+def get_io_datasets(nwbfile):
+    for object_ in nwbfile.objects():
+        if isinstance(object_, TimeSeries) or isinstance(object_, VectorData):
```

Comment:
I didn't think the `VectorData` check would work because technically that's not the object attached to the `nwbfile` container (and thus not enumerated by `nwbfile.objects`)

So I guess the check would be if the object is a child of `DynamicTable` and then enumerate the columns of it in that case?

Reply:
yup, I just fixed this